src/Libraries/Microsoft.Extensions.AI.Abstractions/AudioTranscription/DelegatingAudioTranscriptionClient.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Shared.Diagnostics;
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>
+/// Provides an optional base class for an <see cref="IAudioTranscriptionClient"/> that passes through calls to another instance.
+/// </summary>
+/// <remarks>
+/// This is recommended as a base type when building clients that can be chained in any order around an underlying <see cref="IAudioTranscriptionClient"/>.
+/// The default implementation simply passes each call to the inner client instance.
+/// </remarks>
+public class DelegatingAudioTranscriptionClient : IAudioTranscriptionClient
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="DelegatingAudioTranscriptionClient"/> class.
+    /// </summary>
+    /// <param name="innerClient">The wrapped client instance.</param>
+    protected DelegatingAudioTranscriptionClient(IAudioTranscriptionClient innerClient)
+    {
+        InnerClient = Throw.IfNull(innerClient);
+    }
+
+    /// <inheritdoc />
+    public void Dispose()
+    {
+        Dispose(disposing: true);
+        GC.SuppressFinalize(this);
+    }
+
+    /// <summary>Gets the inner <see cref="IAudioTranscriptionClient" />.</summary>
+    protected IAudioTranscriptionClient InnerClient { get; }
+
+    /// <summary>Provides a mechanism for releasing unmanaged resources.</summary>
+    /// <param name="disposing"><see langword="true"/> if being called from <see cref="Dispose()"/>; otherwise, <see langword="false"/>.</param>
+    protected virtual void Dispose(bool disposing)
+    {
+        if (disposing)
+        {
+            InnerClient.Dispose();
+        }
+    }
+
+    /// <inheritdoc />
+    public virtual AudioTranscriptionClientMetadata Metadata => InnerClient.Metadata;
```

Comment:
We just removed Metadata from IChatClient / IEmbeddingGenerator, and instead rely on the GetService implementation to support providing metadata. We should follow suit here as well.

Reply:
Updated